core/src/test/java/org/jdbi/v3/core/statement/TestDefinedAttributeTemplateEngine.java

```diff
@@ -89,4 +89,10 @@ public void testColonInComment() {
         String sql = "/* comment with : colons :: inside it */ select 1";
```

Comment:
let's add a valid `:abc` parameter in here too, not only invalid bait

Reply:
DefineStatementLexer.g is totally agnostic to colons, so I think this test is a leftover of when we split apart StatementRewriter into TemplateEngine and SqlParser